README.adoc

```diff
@@ -205,17 +205,15 @@ the Zipkin server in a form of a user feature for Open Liberty.
 
 This feature is already configured for you in your `pom.xml` and `server.xml` files. It will be
 downloaded and installed automatically into each service when you run a Maven build. You can find it
-enabled in your `server.xml` files as:
-
-// This feature is already configured for you and it will be installed automatically into each service
-// when you build them. You can find it enabled in your `server.xml` files:
+enabled in your `server.xml` files:
 
 [source, xml, role="no_copy"]
 ----
 <feature>usr:opentracingZipkin-0.30</feature>
 ----
 
-Your `pom.xml` files download and install it using the following Maven plugin:
+The Maven plugin responsible for downloading and installing the feature is:
```

Comment:
We have to have a complete sentence before a colon. Think of it like a period.
Maybe, "The following Maven plugin is responsible for downloading and installing the feature:"

Reply:
yep that works for me